lib/cadet_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,171 @@
+defmodule CadetWeb.AuthController do
+  use CadetWeb, :controller
+
+  import Ecto.Changeset
+
+  use PhoenixSwagger
+
+  alias Cadet.Accounts
+  alias Cadet.Auth.Guardian
+  alias Cadet.Accounts.Form
+
+  def create(conn, %{"login" => attrs}) do
+    changeset = Form.Login.changeset(%Form.Login{}, attrs)
+
+    if changeset.valid? do
+      login = apply_changes(changeset)
+
+      case Accounts.sign_in(login.email, login.password) do
+        {:ok, user} ->
+          {:ok, access_token, _} =
+            Guardian.encode_and_sign(user, %{}, token_type: "access", ttl: {1, :hour})
+
+          {:ok, refresh_token, _} =
+            Guardian.encode_and_sign(user, %{}, token_type: "refresh", ttl: {52, :weeks})
+
+          conn
+          |> render("token.json", access_token: access_token, refresh_token: refresh_token)
+
+        {:error, _reason} ->
+          conn
+          |> send_resp(403, "Wrong email and/or password")
+      end
+    else
+      conn
+      |> send_resp(404, "Missing parameters")
+    end
+  end
+
+  def create(conn, _params) do
+    conn
+    |> send_resp(404, "Missing parameters")
+  end
+
+  def refresh(conn, %{"refresh_token" => refresh_token}) do
+    case Guardian.exchange(refresh_token, "refresh", "access") do
+      {:ok, {refresh_token, _}, {access_token, _}} ->
+        conn
+        |> render("token.json", access_token: access_token, refresh_token: refresh_token)
+
+      {:error, _reason} ->
+        conn
+        |> send_resp(401, "Invalid Token")
+    end
+  end
+
+  def refresh(conn, _params) do
+    conn
+    |> send_resp(404, "Missing parameter(s)")
+  end
+
+  def logout(conn, %{"refresh_token" => refresh_token}) do
+    case Guardian.decode_and_verify(refresh_token) do
+      {:ok, _} ->
+        # Currently does nothing, need to integrate with GuardianDb
+        Guardian.revoke(refresh_token)
+
+        conn
+        |> send_resp(200, "OK")
+
+      {:error, _} ->
+        conn
+        |> send_resp(401, "Invalid Token")
+    end
+  end
+
+  def logout(conn, _params) do
+    conn
+    |> send_resp(404, "Missing parameter(s)")
+  end
+
+  swagger_path :create do
+    post("/auth")
+    summary("Obtain access and refresh tokens to authenticate user")
```

Comment:
Since this is API documentation, this should be more descriptive so the frontend team knows what to do just by reading API docs. For instance, I think you should specify which HTTP header to put the token in, the token expiry time, etc